.env.dev.aws.yaml

```diff
@@ -0,0 +1,25 @@
+VITE_API_BASE_URL: ENC[AES256_GCM,data:yzAevR3tTEfK/ETekvhgPm/9ws6ovFOv9c1K2OvWmKOPs9+EzxQaCGUIXs8PoZqnHu23UOAOFA==,iv:WQkcddTt23MGoFMT1XYKY9NyVgraB6iEJA9UAXAkkck=,tag:2SPwA3NjKHmYVRrlR44dgA==,type:str]
```

Comment:
additionally, I also use the following values: VITE_BASE_OAUTH_STATE=9lu8d7a9; 
VITE_STATE_ID_COOKIE_LIFETIME=86400

Reply:
Added